M3/practica/index.html

```diff
@@ -0,0 +1,355 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>La web del curso</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link type="text/css" rel="stylesheet" href="estilos.css"/>
+    <script src="bower_components/jquery/dist/jquery.min.js"></script>
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.min.css">
+    <script src="bower_components/bootstrap/dist/js/bootstrap.min.js"></script>
+    <script>
+
+        $(document).ready(function () {
+
+            var tablas_cargadas = 0;
+
+            function habilitar_boton() {
+                if (++tablas_cargadas == 2) {
+                    if (clientes_agotados == 1) {
+                        clientes_agotados = 0;
+                    }
+                    tablas_cargadas = 0;
+                    $("#boton-buscar").removeAttr("disabled");
+                }
+            }
+
+            var clientes_agotados = 0;
+
+            function verificar_clientes_agotados() {
+                if (++clientes_agotados == 2) {
+                    clientes_agotados = 0;
+                    mostrar_alert("listado", "agotado");
+                }
+            }
+
+            function mostrar_alert(tab, id) {
+                $("div#alerts-container-" + tab).children("div:not(#" + id + ")").hide().parent().children("div#" + id).show();
```

Comment:
No hace falta que hagas lo del :not 
Podés ocultar todas y después mostrar la que querés (de hecho ya estaba oculta)